wiki/Tournaments/SOFT/4/en.md

```diff
@@ -98,6 +134,147 @@ The schedule can be found [here](https://docs.google.com/spreadsheets/d/e/2PACX-
 
 ## Match Results
 
+### Group Stage
+
+| Saturday, 2019-04-27 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-: |
```

Comment:
```suggestion
| --: | :-: | :-: | :-- | :-- |
```
... I think? I assume you're using a standard format from all tournaments though so maybe not worth changing just for this update